src/components/app/Popouts/Popouts/AddServer.vue

```diff
@@ -141,6 +141,10 @@ export default {
         this.closeMenu();
       }
     },
+    keyDownEvent(event) {
+      if (event.keyCode !== 27) return; // 27 = escape
```

Comment:
the `if isNotKey then stop` pattern is not as understandable as `if IsKey then do { ... }`.
changing this to the second pattern like so would be more readable.
```js
if ( key is pressed) {
  closeMenu
}
```